org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/parser/AbstractCommentParser.java

```diff
@@ -349,9 +348,37 @@ protected boolean commentParse() {
 						lineHasStar = false;
 						// Fix bug 51650
 						this.textStart = -1;
+						// Only reset textStart if not in an unclosed markdown inline tag
+						if (!(this.markdown && this.inlineTagStarted)) {
```

Comment:
Considering the previous line, this is redundant.

Reply:
You are right, ensure to reset the textStart inside the in `if` condition.